docs/pages/guides/view-transition.md

```diff
@@ -0,0 +1,622 @@
+    ```http
+    POST: /api/output/output_1/update
+    Content-Type: application/json
+
+    {
+      "video": {
+        "root": {
+          "type": "view",
+          "background_color_rgba": "#4d4d4dff",
+          "children": [
+            {
+              "type": "rescaler",
+              "id": "rescaler_1",
+              // highlight-start
+              "width": 480,
+              // highlight-end
+              "child": { "type": "input_stream", "input_id": "input_1" },
+            }
+          ]
+        }
+      }
+    }
+    ```
+
+    ```http
+    POST: /api/output/output_1/update
+    Content-Type: application/json
+
+    {
+      "video": {
+        "root": {
+          "type": "view",
+          "background_color_rgba": "#4d4d4dff",
+          "children": [
+            {
+              "type": "rescaler",
+              "id": "rescaler_1",
+              // highlight-start
+              "width": 1280,
+              "top": 0,
+              "left": 0,
+              "transition": { "duration_ms": 2000 },
+              // highlight-end
+              "child": { "type": "input_stream", "input_id": "input_1" },
+            }
+          ]
+        }
+      }
+    }
+    ```
+  </TabItem>
+  <TabItem value="membrane" label="Membrane Framework">
+    ```elixir
+    def handle_setup(ctx, state) do
+      request = %LiveCompositor.Request.UpdateVideoOutput{
+        output_id: "output_1",
+        root: %{
+          type: :view,
+          background_color_rgba: "#4d4d4dff",
+          children: [
+            %{
+              type: :rescaler,
+              id: "rescaler_1",
+              // highlight-start
+              width: 480,
+              // highlight-end
+              child: %{ type: :input_stream, input_id: :input_1 },
+            }
+          ]
+        }
+      }
+      {[
+        notify_child: {:live_compositor, request},
+        start_timer: {:add_videos_timer, Membrane.Time.seconds(2)}
+      ], state}
+    end
+
+    def handle_tick(:start_transition, _ctx, state)
+      request = %LiveCompositor.Request.UpdateVideoOutput{
+        output_id: "output_1",
+        root: %{
+          type: :view,
+          background_color_rgba: "#4d4d4dff",
+          children: [
+            %{
+              type: :rescaler,
+              id: "rescaler_1",
+              // highlight-start
+              width: 1280,
+              top: 0,
+              left: 0,
+              transition: %{ duration_ms: 2000 },
+              // highlight-end
+              child: %{ type: :input_stream, input_id: :input_1 },
+            }
+          ]
+        }
+      }
+      {[
+        notify_child: {:live_compositor, request},
+        stop_timer: :start_transition
+      ], state}
+    end
+    ```
+  </TabItem>
+</Tabs>
+
+As you can see on the resulting stream transition did not happen because the `Rescaler` component
+in the initial scene was using static positioning and after the update it was positioned absolutely.
```

Comment:
```suggestion
As you can see the resulting stream transition did not happen because the `Rescaler` component
in the initial scene was using static positioning and after the update it was positioned absolutely.
```